lib/exq/redis.ex

```diff
@@ -48,10 +48,12 @@ defmodule Exq.Redis do
 
   def keys!(redis, search \\ "*") do
     {:ok, keys} = :eredis.q(redis, ["KEYS", search])
+    keys
```

Comment:
Stats module is using this, so that would need to be changed as well:

https://github.com/optikfluffel/exq/blob/lint/lib/exq/stats.ex#L200

We don't have an tests for that, of course would be great to have as well.